vic5_manuscript.tex

```diff
@@ -286,7 +293,7 @@ \section{Results and Discussion}
 
-  Improving the scaling performance of VIC-5 by adding parallel processing functionality enables VIC-5 to be used for more computationally expensive applications, such as high-resolution modeling studies and coupled earth system models (e.g. RASM) that have shorter coupling intervals (e.g. 20 minutes) than the timesteps that have traditionally been used in VIC (hourly to daily). Fig.~\ref{fig:vic_scaling} demonstrates the scaling performance for the two model configurations. For the simpler model configuration (A), VIC run with MPI shows high throughput, but extending parallelization beyond four to eight nodes (144 to 288 cores) provides little speedup. This behavior is indicative of a program that is I/O limited. For the more complex model configuration (B), VIC has significantly lower throughput, but scales efficiently beyond 32 nodes (1,152 cores). In this case, VIC spends much more time inside the physics code than in the I/O code of the model. For example, a comparison of the timing logs for two simulations that each use 144 MPI tasks shows that configuration A spends 48\% on I/O operations, while configuration B spends only 28\%. In practice, the scaling performance for most parallel VIC applications is predominantly a function of a computer's interconnect and disk speeds.
+  Improving the scaling performance of VIC-5 by adding parallel processing functionality enables VIC-5 to be used for more computationally expensive applications, such as high-resolution modeling studies, sensitivity analyses, and coupled earth system models (e.g. RASM) that have shorter coupling intervals (e.g. 20 minutes) than the timesteps that have traditionally been used in VIC (hourly to daily). Fig.~\ref{fig:vic_scaling} demonstrates the scaling performance for the two model configurations. For the simpler model configuration (A), VIC run with MPI shows high throughput, but extending parallelization beyond four to eight nodes (144 to 288 cores) provides little speedup. This behavior is indicative of a program that is I/O limited. For the more complex model configuration (B), VIC has significantly lower throughput, but scales efficiently beyond 32 nodes (1,152 cores). In this case, VIC spends much more time inside the physics code than in the I/O code of the model. For example, a comparison of the timing logs for two simulations that each use 144 MPI tasks shows that configuration A spends 48\% on I/O operations, while configuration B spends only 28\%. In practice, the scaling performance for most parallel VIC applications is predominantly a function of a computer's interconnect and disk speeds. We expect that VIC's scaling performance would be therefore improved on machines with more performant file systems (e.g. solid-state-disks) or faster interconnects.

```

Comment:
"In this case, VIC spends much more time inside the physics code than in the I/O code of the model." ---> "In this case, the physics code is much more time intensive than the I/O code of the model."